faker/providers/person/yo_NG/__init__.py

```diff
@@ -0,0 +1,268 @@
+from collections import OrderedDict
+
+from .. import Provider as PersonProvider
+
+
+class Provider(PersonProvider):
+    formats_female = (
+        "{{first_name_female}} {{last_name}}",
+        "{{first_name_female}} {{last_name}}",
+        "{{first_name_female}} {{last_name}}",
+        "{{first_name_female}} {{last_name}}",
+        "{{first_name_female}} {{last_name}}-{{last_name}}",
+        "{{prefix_female}} {{first_name_female}} {{last_name}}",
+        "{{prefix_female}} {{first_name_female}} {{last_name}}",
+    )
+
+    formats_male = (
+        "{{first_name_male}} {{last_name}}",
+        "{{first_name_male}} {{last_name}}",
+        "{{first_name_male}} {{last_name}}",
+        "{{first_name_male}} {{last_name}}",
+        "{{first_name_male}} {{last_name}}-{{last_name}}",
+        "{{prefix_male}} {{first_name_male}} {{last_name}}",
+        "{{prefix_male}} {{first_name_male}} {{last_name}}",
+    )
+
+    formats = formats_female + formats_male
+
+    first_names_male = [
```

Comment:
Our [coding guidelines](https://github.com/joke2k/faker/blob/master/docs/coding_style.rst#name-lists) require a source for this data. Can you include the source for these list of names?